packages/contracts/contracts/L1/rollup/ChainStorageContainer.sol

```diff
@@ -6,10 +6,10 @@ import { Lib_Buffer } from "../../libraries/utils/Lib_Buffer.sol";
 import { Lib_AddressResolver } from "../../libraries/resolver/Lib_AddressResolver.sol";
 
 /* Interface Imports */
-import { iOVM_ChainStorageContainer } from "./iOVM_ChainStorageContainer.sol";
+import { IChainStorageContainer } from "./IChainStorageContainer.sol";
```

Comment:
Might be out of scope but we can probably get rid of `IChainStorageContainer`.

Reply:
I'm having a look through obsoleted interfaces and interface usage in general as part of the solidity 0.8 upgrade PR #1417 .